tests/ecmp/test_fgnhg.py

```diff
@@ -173,8 +178,94 @@ def partial_ptf_runner(ptfhost, test_case, dst_ip, exp_flow_count, **kwargs):
             qlen=1000,
             log_file=log_file)
 
+def hash_view_cli_verification(duthost, prefix_list):
+    header = ["FG NHG Prefix", "Next Hop", "Hash buckets"]
+    table = []
+    mode = ""
+    compressed_ipv6_prefixes = [] 
+
+    if ":" in prefix_list[0]:
+        mode = "fgnhg_v6" 
+        for ipv6_prefix in prefix_list:
+            subnet = ipv6_prefix.split("/")[1]
+            ipv6_prefix = ipv6_prefix.split("/")[0]
+            ipv6_prefix = unicode(str(ipaddress.IPv6Address(ipv6_prefix)) + "/" + subnet)
+            compressed_ipv6_prefixes.append(ipv6_prefix)
+        prefix_list = compressed_ipv6_prefixes
+    else:
+        mode = "fgnhg_v4" 
+
+    prefix_list = sorted(prefix_list)
+   
+    for prefix in prefix_list:
+        nh_bucket_dict = {}
+        command = 'redis-cli -n 6 HGETALL "FG_ROUTE_TABLE|' + str(prefix) + '"'
```

Comment:
@kktheballer is there a chance having a SONiC CLI to query this data?

Reply:
sorry I don't get this.

I thought we are trying to verify the SONiC CLI by retrieving manually the data needed to reconstruct and then verify the CLI output

Reply:
@kktheballer seems like you are trying to run a CLI UT in dataplane verification test. Does it make sense? What is the purpose?

Reply:
I think we serve two purposes here:

1. We validate the CLI output
2. We validate that the routes are present as expected in our dataplane verification test